tests/testthat/test-shinytest2-tm_a_pca.R

```diff
@@ -0,0 +1,171 @@
+testthat::test_that("e2e - tm_a_pca: data extract changes output", {
+  skip_if_too_deep(5)
+
+  require(shinytest2)
+
+  data <- within(teal_data(), {
+    require(nestcolor)
+    require(ggplot2)
```

Comment:
Does this work for you?
It's weird that when I try to create an app driver using this it says these packages are not available unless I move these requires outside, close to the `require(shinytest2)`

<img width="1717" alt="Screenshot 2024-04-11 at 2 32 59 PM" src="https://github.com/insightsengineering/teal.modules.general/assets/49812166/1f071daf-5449-460e-903f-49ccba4b130e">

Reply:
Crazy that this package already depends ggplot2. Have no idea why this happens.

Reply:
Related to this. I've just pushed changes that prefix in the module (inside `qenv` as it should be).

I can find a way to avoid this locally, but not on CI.

Reply:
This is an issue: https://github.com/rstudio/shinytest2/issues/381

Reply:
had the same for `tm_a_regression`

Reply:
Fixed for this module